### PR TITLE
attempted assessment

### PR DESCRIPTION
**Pull request by romeoyutig**
# Submitting Week 3 Assessment

### ✅ Completed Files

Mark the following as confirmation that the files in this repository are complete.

- [x] Coding Challenges
- [x] Interview Questions
- [x] React Debugging

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -12,21 +14,50 @@
...
+  })
+})
+
+// // good test 
+// //  ReferenceError: fibFunc is not defined
+
+// // b) Create the function that makes the test pass.
+
+const fibFunc = (num) => {
+  let fibNumbers = [1,1]
```
Comment (TsunamiCode5) [inline code-challenges.test.js:46]: AWESOME!

File: interview-questions.md [hunk 2]
```diff
@@ -10,46 +10,64 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+
+
+Researched answer: There are logic components, and display components. The log lies in the app.js, while the child components are usually calling back on variables in the app.js and used as display components. You can reuse components and make them dynamic using state.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: State values are stored in the app.js while props are passed down onto the children component.
+
+State values are values that need to be updated or changed. props are a way to communicate between components where you can pass down state values and display data. Props are a way to reference data, and must be invoked with a component call.
```
Comment (TsunamiCode5) [inline interview-questions.md:43]: Is this your researched answer just moved up?

File: code-challenges.test.js (hunk 1 above)
Comment (romeoyutig) [reply code-challenges.test.js:46]: Thanks for feedback!

**Review by TsunamiCode5 — APPROVED** [review]
Romeo you did a really good job in this week’s assessment. Your code is looking very clean and logical. You’re using really good vocabulary in your pseudo and that helps solidify your understanding of the terms and concepts. Your interview answers are also very well thought through and you’re able to better explain in your owns words in the researched section to cover anything you briefly touched on in your initial answers. This assessment comprehension level is at Advanced Beginner. You did great work!

File: code-challenges.test.js [hunk 3]
```diff
@@ -64,4 +115,29 @@ const accountTransactions2 = [250, -89, 100, -96]
...
+// ReferenceError: newAccount is not defined
+// good test, function is not yet created.
+// 
+
+
 // b) Create the function that makes the test pass.
+
+const sumAccount = (arr) => {
+  let newAccount = 0
+  return arr.map((value) => {
```
Comment (TsunamiCode5) [inline code-challenges.test.js:135]: This is beautiful!

**Pull request merged**